app/src/main/java/org/simple/clinic/home/overdue/OverdueScreen.kt

```diff
@@ -277,7 +263,26 @@ class OverdueScreen : BaseScreen<
         selectedOverdueAppointments = selectedOverdueAppointments,
         isPatientReassignmentFeatureEnabled = features.isEnabled(PatientReassignment),
         locale = locale,
-    ))
+        onCallPatient = { patientId ->
+          composeUiEvents.onNext(CallPatientClicked(patientId))
+        },
+        onOpenPatientDetails = { patientId ->
+          composeUiEvents.onNext(OverduePatientClicked(patientId))
+        },
+        onToggleAppointmentSelection = { appointmentUuid ->
+          composeUiEvents.onNext(OverdueAppointmentCheckBoxClicked(appointmentUuid))
+        },
+        onSearchButtonClicked = {
+          composeUiEvents.onNext(OverdueSearchButtonClicked)
+        },
+        onToggleSection = { overdueAppointmentSectionTitle ->
+          composeUiEvents.onNext(ChevronClicked(overdueAppointmentSectionTitle))
+        },
+        onPendingFooterClicked = {
+          composeUiEvents.onNext(PendingListFooterClicked)
+        }
```

Comment:
These things can probably be callbacks in the `OverdueAppointmentListItem` composable? 🤔